### PR TITLE
Drafts design review second batch

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Fine tune view controller transition and other smaller changes.

### CODE REVIEW

**Comment by vytis** [comment]
Merge in develop pls!:)

**Comment by daehn** [comment]
@vytis done!

File: Wire-iOS/Sources/UserInterface/ConversationList/ComposeEntryViewController.swift [hunk 1]
```diff
@@ -381,13 +373,32 @@ import Cartography
 
 // MARK: Helper
 
+fileprivate extension UIView {
 
-fileprivate extension CGAffineTransform {
+    static func animate(withGroup group: DispatchGroup, duration: TimeInterval, delay: TimeInterval, options: UIViewAnimationOptions, animations: @escaping () -> Void) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/ConversationList/ComposeEntryViewController.swift]: Could be useful in some general-level UIView extension.

**Comment by mikeger** [comment]
@daehn computer says: "Your build ran 485 tests with 12 failures"

**Comment by daehn** [comment]
@mikeger thanks, will have a look!

File: Wire-iOS/Sources/UserInterface/ConversationList/ComposeEntryViewController.swift (hunk 1 above)
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/ConversationList/ComposeEntryViewController.swift]: Good point, will extract it and make it visible in the whole app.

**Comment by vytis** [comment]
BTW you can find failing snapshots in build artifacts tab on CircleCI

**Comment by daehn** [comment]
giphy-5

**Comment by daehn** [comment]
@mikeger @vytis updated, but lets wait until the machines have verified it works.

**Comment by daehn** [comment]
@mikeger @vytis tests are ✅.

**Pull request merged**